### PR TITLE
Integration test for adding a new StableToken

### LINKED ISSUE #7224: Integration test for adding a new StableToken via governance
### Expected Behavior

There is an integration test for adding a new StableToken via governance.

### Current Behavior

No such test.

**Reason for need:** Need standardized tests to run on our next StableToken #3.

### PR DESCRIPTION
**Pull request by asaj**
### Description

This PR adds an integration test for adding a new StableToken, mimicking the planned process for deployment of cEUR, and future additional stable assets.

### Other changes

None.

### Tested

These are tests!

### Related issues

- Fixes #7224 

### Backwards compatibility

Yes, just tests.

### CODE REVIEW

File: packages/protocol/test/common/integration.ts [hunk 1]
```diff
@@ -564,3 +572,187 @@ Array.from([
     })
   })
 )
+
+// TODO: DO NOT FORGET TO REMOVE THE ONLYS HERE
```
Comment (nambrot) [inline packages/protocol/test/common/integration.ts]: We should address this comment and then remove it?

File: packages/protocol/test/common/integration.ts [hunk 2]
```diff
@@ -564,3 +572,187 @@ Array.from([
...
+              // @ts-ignore
+              registry.contract.methods.transferOwnership(accounts[0]).encodeABI()
+            ),
+            'hex'
+          ),
+        }
+      })
+    )
+    await governance.propose(
+      proposalTransactions.map((x: any) => x.value),
```
Comment (nambrot) [inline packages/protocol/test/common/integration.ts]: Is the any necessary? Shouldn't it be typed from above?

Comment (martinvol) [reply packages/protocol/test/common/integration.ts]: I am not sure what you mean, this is basically a copy&paste from here https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/test/common/integration.ts#L153

Comment (yorhodes) [reply packages/protocol/test/common/integration.ts]: this should be used in favor of this entire before hook https://github.com/celo-org/celo-monorepo/blob/a4fe8d89b05b67dd889a6c79d2dc1056c6f4a037/packages/protocol/lib/test-utils.ts#L435

Comment (martinvol) [reply packages/protocol/test/common/integration.ts]: Thanks @yorhodes, will refactor

**Review by yorhodes — APPROVED** [review]

**Pull request merged**